src/Uno.UI/UI/Xaml/UIElement.Pointers.iOS.cs

```diff
@@ -15,6 +16,48 @@ namespace Windows.UI.Xaml
 {
 	partial class UIElement
 	{
+		private class NativePointerId
+		{
+			private static readonly Dictionary<IntPtr, NativePointerId> _instances = new Dictionary<IntPtr, NativePointerId>();
+			private static uint _nextAvailablePointerId;
+
+			private readonly IntPtr _nativeId;
+			private readonly HashSet<UIElement> _leases = new HashSet<UIElement>();
+
+			public uint Value { get; }
+
+			private NativePointerId(IntPtr nativeId)
+			{
+				_nativeId = nativeId;
+				Value = _nextAvailablePointerId++;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this assignment of '_nextAvailablePointerId' or initialize it statically.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5398047)